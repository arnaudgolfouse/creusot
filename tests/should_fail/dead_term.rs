extern crate creusot_contracts;
use creusot_contracts::*;

#[logic(opaque)]
pub fn works() {
    dead
}

#[logic]
pub fn fails1() {
    dead
}

#[trusted]
#[logic]
pub fn fails2() {
    dead
}
