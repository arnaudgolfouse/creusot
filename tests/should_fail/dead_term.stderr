error: The `dead` term can only be used for the body of `logic(opaque)` functions
  --> dead_term.rs:11:5
   |
11 |     dead
   |     ^^^^

error: aborting due to 1 previous error

